Traceback (most recent call last):
  File "/Users/cghiaus/opt/anaconda3/lib/python3.9/site-packages/jupyter_cache/executors/utils.py", line 58, in single_nb_execution
    executenb(
  File "/Users/cghiaus/opt/anaconda3/lib/python3.9/site-packages/nbclient/client.py", line 1112, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
  File "/Users/cghiaus/opt/anaconda3/lib/python3.9/site-packages/nbclient/util.py", line 74, in wrapped
    return just_run(coro(*args, **kwargs))
  File "/Users/cghiaus/opt/anaconda3/lib/python3.9/site-packages/nbclient/util.py", line 53, in just_run
    return loop.run_until_complete(coro)
  File "/Users/cghiaus/opt/anaconda3/lib/python3.9/asyncio/base_events.py", line 642, in run_until_complete
    return future.result()
  File "/Users/cghiaus/opt/anaconda3/lib/python3.9/site-packages/nbclient/client.py", line 553, in async_execute
    await self.async_execute_cell(
  File "/Users/cghiaus/opt/anaconda3/lib/python3.9/site-packages/nbclient/client.py", line 857, in async_execute_cell
    self._check_raise_for_error(cell, exec_reply)
  File "/Users/cghiaus/opt/anaconda3/lib/python3.9/site-packages/nbclient/client.py", line 760, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply_content)
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
[As, Bs, Cs, Ds, us] = dm4bem.tc2ss(TC)
------------------

[0;31m---------------------------------------------------------------------------[0m
[0;31mValueError[0m                                Traceback (most recent call last)
[0;32m/var/folders/vp/k617ngs97xq4vlzlj5ts6sch0000gn/T/ipykernel_3839/3506208553.py[0m in [0;36m<module>[0;34m[0m
[0;32m----> 1[0;31m [0;34m[[0m[0mAs[0m[0;34m,[0m [0mBs[0m[0;34m,[0m [0mCs[0m[0;34m,[0m [0mDs[0m[0;34m,[0m [0mus[0m[0;34m][0m [0;34m=[0m [0mdm4bem[0m[0;34m.[0m[0mtc2ss[0m[0;34m([0m[0mTC[0m[0;34m)[0m[0;34m[0m[0;34m[0m[0m
[0m
[0;32m~/Personel/Py/git_book/dm4bem_book/tutorials/dm4bem.py[0m in [0;36mtc2ss[0;34m(TC)[0m
[1;32m   1336[0m [0;34m[0m[0m
[1;32m   1337[0m     [0;31m# DAE (Differential Algebraic Equations), Eq.(13)[0m[0;34m[0m[0;34m[0m[0m
[0;32m-> 1338[0;31m     [0mK[0m [0;34m=[0m [0;34m-[0m[0mA[0m[0;34m.[0m[0mT[0m [0;34m@[0m [0mG[0m [0;34m@[0m [0mA[0m[0;34m[0m[0;34m[0m[0m
[0m[1;32m   1339[0m     [0mKb[0m [0;34m=[0m [0mA[0m[0;34m.[0m[0mT[0m [0;34m@[0m [0mG[0m[0;34m[0m[0;34m[0m[0m
[1;32m   1340[0m [0;34m[0m[0m

[0;32m~/opt/anaconda3/lib/python3.9/site-packages/pandas/core/frame.py[0m in [0;36m__matmul__[0;34m(self, other)[0m
[1;32m   1490[0m         [0mMatrix[0m [0mmultiplication[0m [0musing[0m [0mbinary[0m[0;31m [0m[0;31m`[0m[0;34m@[0m[0;31m`[0m [0moperator[0m [0;32min[0m [0mPython[0m[0;34m>=[0m[0;36m3.5[0m[0;34m.[0m[0;34m[0m[0;34m[0m[0m
[1;32m   1491[0m         """
[0;32m-> 1492[0;31m         [0;32mreturn[0m [0mself[0m[0;34m.[0m[0mdot[0m[0;34m([0m[0mother[0m[0;34m)[0m[0;34m[0m[0;34m[0m[0m
[0m[1;32m   1493[0m [0;34m[0m[0m
[1;32m   1494[0m     [0;32mdef[0m [0m__rmatmul__[0m[0;34m([0m[0mself[0m[0;34m,[0m [0mother[0m[0;34m)[0m[0;34m:[0m[0;34m[0m[0;34m[0m[0m

[0;32m~/opt/anaconda3/lib/python3.9/site-packages/pandas/core/frame.py[0m in [0;36mdot[0;34m(self, other)[0m
[1;32m   1444[0m             [0mcommon[0m [0;34m=[0m [0mself[0m[0;34m.[0m[0mcolumns[0m[0;34m.[0m[0munion[0m[0;34m([0m[0mother[0m[0;34m.[0m[0mindex[0m[0;34m)[0m[0;34m[0m[0;34m[0m[0m
[1;32m   1445[0m             [0;32mif[0m [0mlen[0m[0;34m([0m[0mcommon[0m[0;34m)[0m [0;34m>[0m [0mlen[0m[0;34m([0m[0mself[0m[0;34m.[0m[0mcolumns[0m[0;34m)[0m [0;32mor[0m [0mlen[0m[0;34m([0m[0mcommon[0m[0;34m)[0m [0;34m>[0m [0mlen[0m[0;34m([0m[0mother[0m[0;34m.[0m[0mindex[0m[0;34m)[0m[0;34m:[0m[0;34m[0m[0;34m[0m[0m
[0;32m-> 1446[0;31m                 [0;32mraise[0m [0mValueError[0m[0;34m([0m[0;34m"matrices are not aligned"[0m[0;34m)[0m[0;34m[0m[0;34m[0m[0m
[0m[1;32m   1447[0m [0;34m[0m[0m
[1;32m   1448[0m             [0mleft[0m [0;34m=[0m [0mself[0m[0;34m.[0m[0mreindex[0m[0;34m([0m[0mcolumns[0m[0;34m=[0m[0mcommon[0m[0;34m,[0m [0mcopy[0m[0;34m=[0m[0;32mFalse[0m[0;34m)[0m[0;34m[0m[0;34m[0m[0m

[0;31mValueError[0m: matrices are not aligned
ValueError: matrices are not aligned

